CMakeLists.txt

```diff
@@ -36,7 +36,7 @@ set(IGN_MATH_VER ${ignition-math7_VERSION_MAJOR})
 #--------------------------------------
 # Find ignition-common
 ign_find_package(ignition-common5 REQUIRED
-  COMPONENTS graphics events)
```

Comment:
The geospatial component in `ign-common` is optional, while `geospatial` is required here. Is that intended?

Reply:
hmm if we make the `geospatial` component optional here, we will need to make heightmaps in ign-rendering optional as well and only build it if the `geospatial` component is available. We'll likely need to do the same in ign-gazebo.

Reply:
should we revert the ign-common PR for now while we sort this out?

Reply:
ok that sounds good as this will require a bigger change

Reply:
https://github.com/ignitionrobotics/ign-common/pull/291

Reply:
In a previous f2f meeting with @chapulina , we decided against making `GDAL` an optional component because it was going to add a layer of difficulty managing

Reply:
Yeah I remember also discussing this at the weekly meeting. GDAL will be required for heightmaps. Ignition Physics already has a separate heightmap component, so only that component needs to depend on `geopspatial`. But Ignition Rendering doesn't have that separation and with the current architecture I think it may be difficult to break that apart.